src/autocomplete.ts

```diff
@@ -101,6 +143,15 @@ export default class Autocomplete {
     }
   }
 
+  updateFeedbackForScreenReaders(input: ScreenReaderAccouncementConfig): void {
+    setTimeout(() => {
+      if (this.feedback) {
+        this.feedback.innerHTML = getAnnouncementStringByEvent(input)
+        this.container.dispatchEvent(new CustomEvent('sr-update'))
+      }
+    }, 100)
```

Comment:
Would you mind detailing why we need the timeout, and why `100` was chosen?   I'm guessing it's an order-of-events issue, but those can usually be solved with `0` timeout.

Reply:
I suspect this will change after our screenreader auditors touch it, but it allows time for the announcements of other events to finish. If "polite" is used instead, VoiceOver ignores the announcement half the time.

Reply:
This actually got me thinking about how we'll test with our auditors. Little change incoming.